.github/workflows/ci.yml

```diff
@@ -167,10 +167,13 @@ jobs:
         task:
           - name: Check Newsfragment
             tox: check-newsfragment
+            run-if: ${{ github.head_ref != 'pre-commit-ci-update-config' }}
           - name: Check package manifest
             tox: check-manifest
+            run-if: true
           - name: Check pre-commit
             tox: pre-commit
+            run-if: true
```

Comment:
```suggestion
```

I don't think this is necessary at all, since we have pre-commit.ci. It's waste of time and resources?

Reply:
I don't know what to say.

The `tox pre-commit` can be helpful for local testing, and this CI jobs makes sure the local helper for pre-commit works.

Reply:
This is not tox, this is GitHub Actions. 100% keep it in tox.ini – there's just no point to run it in CI twice.

Reply:
(i see your point to verify it works, but it's imho not worth it)